### ISSUE TITLE
opensubtitles-api\index.js:58 Uncaught (in promise) Error: 401 Unauthorized

### ISSUE THREAD

**Issue by adrian-herscu** [popcorntime/popcorntime#1264]
Operating System Version:
Windows 10

Popcorn Time Version:
0.4.3

Download date:
April 4th 2020

#### Expected Behaviour
Download and render subtitles.

#### Actual Behaviour
The subtitles control is on "Disabled" state.

#### Steps to reproduce the behaviour

1. Clean install
2. Setup OpenSubtitles account and select default subtitles language
3. Choose a movie which previously seen with subtitles
4. Click the "subtitle" control (appears as an "X") --> first error emitted in console
5. Click "Watch Now"
6. No subtitles and there is only Disabled/Custom options

Here are the console logs:
```
app.js:17 [INFO] Show movie detail (tt0114069)
app.js:17 [INFO] hideunused (tt0114069)
app.js:17 [INFO] Returning cached provider OpenSubtitles
lang_dropdown.js:48 en
play_control.js:194 Audios: en
quality-selector.js:79 Select quality:  720p
play_control.js:182 Subtitles: none
C:\Users\adrian\AppData\Local\Popcorn-Time\node_modules\opensubtitles-api\index.js:58 Uncaught (in promise) Error: 401 Unauthorized
    at C:\Users\adrian\AppData\Local\Popcorn-Time\node_modules\opensubtitles-api\index.js:58:19
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(anonymous) @ C:\Users\adrian\AppData\Local\Popcorn-Time\node_modules\opensubtitles-api\index.js:58
processTicksAndRejections @ internal/process/task_queues.js:97
app.js:17 [INFO] Returning cached provider OpenSubtitles
app.js:17 [INFO] Loading torrent
app.js:17 [INFO] Loading torrent: startingDownload
app.js:17 [INFO] Loading torrent: waitingForSubtitles
app.js:36 [ERROR] subtitleProvider.fetch() Error: 401 Unauthorized
    at C:\Users\adrian\AppData\Local\Popcorn-Time\node_modules\opensubtitles-api\index.js:58:19
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
win.error @ app.js:36
(anonymous) @ streamer.js:599
processTicksAndRejections @ internal/process/task_queues.js:97
app.js:17 [INFO] Loading torrent: ready
app.js:17 [INFO] Watching: Outbreak
app.js:17 [INFO] Player closed
app.js:17 [INFO] Streaming cancelled
```

**Comment by aretsmarvin** [comment]
I've the same issue (yesterday it worked fine).

**Comment by kiriles90** [comment]
Duplicate https://github.com/popcorn-official/popcorn-desktop/issues/1252 and fixed in https://ci.popcorntime.app/job/Popcorn-Time-Desktop/407/

(you still need to login, this depends on opensubtitles if they ever lift this 'everyone needs to login for the API' limitation again)

**Comment by dutchy1001** [comment]
Fix explained in #1318

**Comment by kiriles90** [comment]
@Persei08 issue is fixed, close please
